scrapyd_k8s/api.py

```diff
@@ -25,7 +25,23 @@ def healthz():
 
 @app.get("/daemonstatus.json")
 def api_daemonstatus():
-    return { "status": "ok", "spiders": 0 }
+    node_name = config.scrapyd().get("node_name", launcher.get_node_name())
+    total_pending = 0
+    total_running = 0
+    total_finished = 0
+    projects = config.listprojects()
+    for project_id in projects:
+        jobs = launcher.listjobs(project_id)
```

Comment:
Why do you iterate over the projects? It would be much lighter to query all jobs in one go, without the project filter.

Reply:
Yeah, I thought that jobs can be divided across projects and if I won't check all the jobs of all the projects, I might miss something, I guess we can just simply access all the jobs